mods/ts/rules/infantry.yaml

```diff
@@ -268,15 +257,10 @@ JUMPJET:
 
 CHAMSPY:
 	Inherits: ^Infantry
-	Buildable:
-		Queue: Infantry
-		BuildPaletteOrder: 60
-		Prerequisites: ~nahand, natech
 	Valued:
 		Cost: 700
 	DisguiseToolTip:
 		Name: Chameleon Spy
-		Description: Infiltrates enemy structures to gather \nintelligence. Exact effect depends on the \nbuilding infiltrated.\n  Strong vs Nothing\n  Weak vs Everything\n  Special Ability: Disguised
```

Comment:
I guess pchote removed this since the spy isn't buildable, but I agree that there would be no harm in keeping this, saves us the hassle of adding it back if we ever decide to make the spy buildable.